flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/decorators/JavaCmdTaskManagerDecorator.java

```diff
@@ -52,7 +49,7 @@ public JavaCmdTaskManagerDecorator(KubernetesTaskManagerParameters kubernetesTas
 	public FlinkPod decorateFlinkPod(FlinkPod flinkPod) {
 		final Container mainContainerWithStartCmd = new ContainerBuilder(flinkPod.getMainContainer())
 			.withCommand(kubernetesTaskManagerParameters.getContainerEntrypoint())
-			.withArgs(Arrays.asList(NATIVE_KUBERNETES_COMMAND, getTaskManagerStartCommand()))
+			.withArgs(KubernetesUtils.getStartCommandWithBashWrapper(getTaskManagerStartCommand()))
```

Comment:
Same here. `KubernetesUtils#getStartCommandWithBashWrapper` can be moved into `getTaskManagerStartCommand`.